auctions/english-auction/project/auction-contract/tests/functions/bid.rs

```diff
@@ -653,7 +653,9 @@ mod revert {
     }
 
     #[tokio::test]
-    #[should_panic(expected = "Revert(18446744073709486080)")]
+    #[ignore]
+    // TODO: test is not set up to hit the error properly
+    // #[should_panic(expected = "NFTTransferNotApproved")]
```

Comment:
While this test does panic as expected with `#[should_panic(expected = "Revert(18446744073709486080)")]`, I understand not wanting some long random number used. It may be better to not ignore the test and instead leave a comment linking an issue as to why `#[should_panic(expected = "NFTTransferNotApproved")]` can't be used.

Reply:
Ignoring the test makes it immediately pop up as an issue when you run the tests locally.
On the other hand if the test fails for any other reason then it may be something we want to look at more carefully.

The reason that I am convinced by is that project management on GH via issues superseds someone actually noticing the tests being ignored if they are run locally and knowing to actually look into that or ask someone about it. If there is an issue then anyone that watches the repo will be aware of this and if the test fails for any other reason than that terrible string then we will immediately know that we have to address it.

I will make adjustments and raise an issue.

Reply:
#330